<p><code>calc</code> is a CSS3 function that provides the possibility to do simple math in CSS (add, subtract, divide, multiply). Without spaces
around operators, <code>calc</code> will have no effect.</p>
<p>More precisely, before an operator, there must be a single whitespace or a newline plus indentation. After an operator, there must be a single
whitespace or a newline.</p>
<h2>Noncompliant Code Example</h2>
<pre>
#div1 {
    position: absolute;
    width: calc(100%- 100px); /* Noncompliant; no space after the % sign */
}
</pre>
<h2>Compliant Solution</h2>
<pre>
#div1 {
    position: absolute;
    width: calc(100% - 100px);
}
</pre>

