<p>Leftover empty blocks are usually introduced by mistake. They are useless and prevent readability of the code. They should be removed or completed
with real code.</p>
<h2>Noncompliant Code Example</h2>
<pre>
a { }
</pre>
<h2>Compliant Solution</h2>
<pre>
a { color: pink; }
</pre>

