<p>The W3C specifications define the valid pseudo-class selectors. Only the official and browser-specific pseudo-class selectors should be used to get
the expected impact in the final rendering.</p>
<h2>Noncompliant Code Example</h2>
<pre>
a:hoverr { /* Noncompliant; there is a typo on the word "hover" */
...
}
</pre>
<h2>Compliant Solution</h2>
<pre>
a:hover {
...
}
</pre>

