<p>According to the W3C specifications:</p>
<blockquote>
  <p>A string cannot directly contain a newline. To include a newline in a string, use an escape representing the line feed character in ISO-10646
  (U+000A), such as "\A" or "\00000a".</p>
  <p>[...]</p>
  <p>It is possible to break strings over several lines, for aesthetic or other reasons, but in such a case the newline itself has to be escaped with
  a backslash (\).</p>
</blockquote>
<h2>Noncompliant Code Example</h2>
<pre>
a {
  content: "first
    second";
}
</pre>
<h2>Compliant Solution</h2>
<pre>
a {
  content: "first\Asecond";
}
</pre>
<h2>See</h2>
<ul>
  <li> <a href="https://www.w3.org/TR/CSS2/syndata.html#strings">CSS Specification</a> - Strings </li>
</ul>

