<p>The W3C specifications say comments should be defined using <code>/* ... */</code>. The use of <code>//</code> is not supported on all browsers and
can lead to unexpected results.</p>
<h2>Noncompliant Code Example</h2>
<pre>
a { // color: pink; }
</pre>
<h2>Compliant Solution</h2>
<pre>
a { /* color: pink; */ }
</pre>

