<p>The W3C specifications define the valid <code>at-rules</code>. Only the official and browser-specific <code>at-rules</code> should be used to get
the expected impact in the final rendering.</p>
<h2>Noncompliant Code Example</h2>
<pre>
@unknown {}
</pre>
<h2>Compliant Solution</h2>
<pre>
@media (max-width: 960px) {}
</pre>

