<p>The W3C specifications define the valid pseudo-element selectors. Only the official and browser-specific pseudo-element selectors should be used to
get the expected impact in the final rendering.</p>
<h2>Noncompliant Code Example</h2>
<pre>
a::beforre { /* Noncompliant; there is a typo on the word "before" */
...
}
</pre>
<h2>Compliant Solution</h2>
<pre>
a::before {
...
}
</pre>

