<p>The W3C specifications define the valid media features. Only the official and browser-specific media features should be used to get the expected
impact in the final rendering.</p>
<h2>Noncompliant Code Example</h2>
<pre>
@media screen and (unknown: 1000px) { .. }
</pre>
<h2>Compliant Solution</h2>
<pre>
@media screen and (width: 1000px) { .. }
</pre>
<h2>See</h2>
<ul>
  <li> <a href="https://www.w3schools.com/cssref/css3_pr_mediaquery.asp">CSS Specification</a> - <code>@media</code> rule </li>
</ul>

