<p>HTML, SVG, and MathML define the selectors which can be used in a CSS. A selector that is not part of them is likely to be a typo or a
misunderstanding of the CSS syntax.</p>
<h2>Noncompliant Code Example</h2>
<pre>
field {}

ul list {}
</pre>
<h2>Compliant Solution</h2>
<pre>
input {}

ul li {}
</pre>

